structural-analysis-elements/structuralcurvemember.md

```diff
@@ -1,384 +1,53 @@
+|                    Nodes                    |      String      |                                                                                        N2; N3; N4                                                                                       |         yes        | <p>All nodes that belong to the curve member and define its geometric shape.</p><p>The names of the nodes are separated by ; (semicolon) and space. The order of the nodes has to be from beginning to end.</p>                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|                   Segments                  |      String      |                                                                    Line; Circular Arc; Bezier; Parabolic arc; Spline                                                                    |         yes        | Defines the shape of the curve between two next nodes. The names are separated by ; (semicolon) and space.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|                  Begin node                 |      String      |                                                                                            N1                                                                                           |         no         | <p>The starting node of the 1D member.</p><p>Has to be specified in the StructuralPointConnection sheet</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I'd state here that begin node should be equal to the first node from the nodes column.
Maybe even state that this column is only here for filtering purposes when viewing the SAF file in Excel? It doesn't have any benefit otherwise